.github/contributing.md

```diff
@@ -34,74 +34,37 @@ Discussions are where we have conversations about Rafiki.
 If you would like to discuss topics about the broader ecosystem, have a new idea, or want to show off your work - join us in [discussions](https://github.com/interledger/rafiki/discussions).
 
 ### :beetle: Issues
-We use Github issues to track tasks that contributors can help with. We haven't finalised labels yet for contributors to tackle. If you want to help with work related to an issue, please comment on the issue before starting work on it.
+We use GitHub issues to track tasks that contributors can help with. We haven't finalized labels yet for contributors to tackle. If you want to help with work related to an issue, please comment on the issue before starting work on it.
 
-If you've found something that needs fixing, search open issues to see if someone else has reported the same thing. If it's something new, open an issue. We'll use the issue to have a conversation about the problem you want to fix.
+If you've found something that needs fixing, search open issues to see if someone else has reported the same thing. If it's something new, open an issue. We'll use the issue to discuss the problem you want to fix.
 
 ### :hammer_and_wrench: Pull requests
 Feel free to fork and create a pull request on changes you think you can contribute.
 
 The team will review your pull request as soon as possible.
 
-## Working in the rafiki repository
-
-This project uses yarn 2. We use [zero-installs](https://yarnpkg.com/features/zero-installs)
-which means you won't need to install dependencies when you clone the repo.
-This does have [security implications](https://yarnpkg.com/features/zero-installs#does-it-have-security-implications)
-that are checked in CI.
+### :books: Documentation
+We maintain documentation for Rafiki on [rafiki.dev](https://github.com/interledger/rafiki.dev). A list of issues being tracked across the Interledger ecosystem (including rafiki) is maintained in the [Documentation project](https://github.com/orgs/interledger/projects/5/views/1).
```

Comment:
Should rafiki.dev be included here even if it's still quite fresh?

Reply:
I'll reword that to be clear it's still fresh.